tests/shipping.test.js

```diff
@@ -299,6 +300,41 @@ describe('Shipping', () => {
         addresses.pickupPointAddress.receiverName
       )
     })
+
+    it('should return the deliveryWindow for null slas (slas array is null in fulfillment)', () => {
```

Comment:
This new test is a good example that proves this method not always work! 😄 
All other tests are consistent and should not be changed in my opinion! 😄

Reply:
Agreed!